source/text/fontloader.ts

```diff
@@ -47,23 +47,23 @@ export class FontLoader {
             return;
         }
 
-        const values = pairs.get('padding')!.split(',');
+        const attributes = pairs.get('padding')!.split(',');
```

Comment:
@cgcostume So in the end you renamed the other `values` :D
But these here are actually padding values, not really attributes...

Reply:
focus! please! 😊